rpmbuild/mock-custom-build.cfg.j2

```diff
@@ -2,9 +2,12 @@ include('/etc/mock/{{ chroot }}.cfg')
 config_opts['rpmbuild_networking'] = True
 config_opts['use_host_resolv'] = True
 
+{# See https://github.com/fedora-copr/copr/issues/3140 #}
+{%- if not chroot.startswith("fedora-rawhide-") %}
```

Comment:
We really need `"fedora-40" in chroot` condition here.  Because of https://github.com/rpm-software-management/mock/pull/1332